data/transportation/van-go.yaml

```diff
@@ -1,6 +1,7 @@
 name: Van-GO!
 category: Bus
 url: http://van-go.info/schedule/
-synopsis: Van-GO is stopped due to snow. Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
+synopsis: Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
 description:
-  Van-GO is stopped today due to snow. Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
+  Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use
+  only.
```

Comment:
should this be on a new line?

Reply:
That's how it wraps

Reply:
It wouldn't have a line number, would it?

Reply:
Corrent. I wrap these at 80 columsn.

Reply:
Alright :+1: